### DISCUSSION TITLE
在openwrt上挂载阿里云时报错

### DISCUSSION THREAD

**Discussion by Cykaihappy** [AlistGo/alist#4602] [Q&A]
报错：Failed get storage driver: no mount path for an storageis: /aliyun
好像不止这一个，其他的网盘挂载时也会这样，但是本地的就没有
这个文件夹我是创建了的，而且没有挂载其他的，换了几个其他的文件夹都不可以，求助

**Comment by anwen-anyi** [comment]
盲猜你升级后版本被降低了？ 你看看你现在是什么版本~

**Comment by wzhisaso** [comment]
+1,一样的问题
